code/python/tools/dataset_download_images.py

```diff
@@ -9,28 +9,20 @@
 parser = argparse.ArgumentParser()
 parser.add_argument("--downloads_dir", required=True)
 parser.add_argument("--decompress_dir")
+parser.add_argument("--delete_archive_after_decompression", action="store_true")
 args = parser.parse_args()
 
-
```

Comment:
For consistency with the rest of the codebase, can you please make it so there is one new-line here or three, but not two. Otherwise this code is ready to merge. Thank you 😀